bundle-workflow/src/test_workflow/bwc_test/bwc_test_suite.py

```diff
@@ -9,33 +9,43 @@
 
 import logging
 import os
-import subprocess
 
+from paths.script_finder import ScriptFinder
+from system.execute import execute
 from test_workflow.test_component import TestComponent
 
 
 class BwcTestSuite:
     manifest: str
+    work_dir: str
     component: str
     keep: bool
 
     def __init__(self, manifest, work_dir, component=None, keep=False):
         self.manifest = manifest
         self.work_dir = work_dir
+        self.script_finder = ScriptFinder()
         self.component = component
         self.keep = keep
 
-    def run_tests(self, work_dir):
-        bwc_script = "bwctest.sh"
-        run_bwctests = f"./{bwc_script}"
-        output = subprocess.check_output(run_bwctests, cwd=work_dir, shell=True)
-        return output
+    def run_tests(self, work_dir, component_name):
+        script = self.script_finder.find_bwc_test_script(
+            component_name, work_dir
+        )
+        if os.path.exists(script):
+            cmd = f"{script}"
+            (status, stdout, stderr) = execute(cmd, work_dir, True, False)
+            return (status, stdout, stderr)
+        else:
```

Comment:
Is this possible? There's a default one, no?

Reply:
Lol thats true. That was just incase to understand where things are failing. 
My likely a no-op.

Reply:
You should remove the `if`. Execute will fail if you don't have the script anyway.

Reply:
Sure, that makes sense. Let me quickly fix that.